src/components/emoji/Emoji.tsx

```diff
@@ -1,26 +0,0 @@
-import { emojiFileNames } from 'src/utils/emojiFileNames';
-import styled from 'styled-components';
-
-const StyledImage = styled.img`
-  vertical-align: text-top;
-  height: 20px;
-  width: auto;
-  display: inline-block;
-`;
-
-type Props = {
-  code: string;
-};
-
-export const Emoji = ({ code }: Props) => {
-  const emojiFileName = emojiFileNames.find(({ emoji }) => code === emoji);
-
-  if (emojiFileName) {
-    const attribution = `Attribution for Twitter Emojis: ©️ Twitter, Inc.`;
-    const alt = `${emojiFileName.emoji} ${attribution}`;
-    return <StyledImage alt={alt} src={`/emojis/${emojiFileName.fileName}`} />;
-  }
```

Comment:
It looks like we have folder `emoji` here. Do we also need to remove this folder too? 🤔

Reply:
No, it is actually an external reference (for the consuming project), hence why it didn't make any sense  to be here.